db/migrate/20230116174207_create_pg_search_documents.rb

```diff
@@ -0,0 +1,17 @@
+class CreatePgSearchDocuments < ActiveRecord::Migration[7.0]
+  def up
+    say_with_time("Creating table for pg_search multisearch") do
+      create_table :pg_search_documents do |t|
```

Comment:
what is this table for?

Reply:
It's required by the [`pg_search` gem](https://github.com/Casecommons/pg_search#multi-search) to support searching in multiple fields/tables at the same time

> Before using multi-search, you must generate and run a migration to create the pg_search_documents database table.

I was able to fix the indexing logic and get all the documents generated in this table
<img width="689" alt="image" src="https://user-images.githubusercontent.com/26656167/218317033-15ee0dac-0d78-4e78-bad4-953a7669f055.png">